pyiron_continuum/fenics/job/generic.py

```diff
@@ -41,7 +36,7 @@
 __date__ = "Dec 6, 2020"
 
 
-class Fenics(GenericJob):
+class Fenics(TemplateJob):
```

Comment:
Is there a particular reason that you use `TemplateJob` instead of `GenericJob`? As you know I'm a fierce opponent of `TemplateJob`, especially when it comes to a class that's going to be included inside pyiron

Reply:
I directly leverage input as a DatContainer. GenericJob is not updated for that yet, is it?

I think I'm the other way around -- I fiercely oppose GenericJob, TemplateJob should just get that name 😂 Anyhow, once these aren't separate things I am extremely happy to rebase

Reply:
ok in this regard we're on the same page. As soon as we can get rid of `TemplateJob`, I'll change it here, which hopefully will happen within a few months.

Reply:
But then do we still need to import `DataContainer` in this file?

Reply:
We probably don't strictly. Right now it only appears in the type hint for `input` and `output` in the job's docstring. In principle we could get rid of the import, but PyCharm sees it in the docstring and thus doesn't flag the import as "unused" -- ok, I just tested and actually because the import is there I can CTRL+click on the type in the docstring and the IDE takes me straight to the definition! So let's leave it.